ddtrace/debugging/_debugger.py

```diff
@@ -223,6 +225,8 @@ def enable(cls) -> None:
 
         core.dispatch("dynamic-instrumentation.enabled")
 
+        debugger._load_local_config()
```

Comment:
We should probably make sure the debugger is initialised fully before dispatching the enabled event. Not a big deal in this case, but it might make the code more future-proof. A good way to ensure this would be to move this call inside `Debugger.__init__`.